Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -108,6 +111,32 @@ extension ServiceRunnable {
         headers.append(.timestamp(value: timestamp))
         headers.append(.sourceSDK(value: "iOS"))
         headers.append(.sourceSDKVersion(value: SmileID.version))
+        let timestamp = Date().toISO8601WithMilliseconds()
```

Comment:
util func? I see it used in multiple places

Reply:
This is an extension to the Data object. Don't think we need an extra util function for that. Extensions are the way to do those things on iOS side as far as I am aware.